placement-rules-in-sql.md

```diff
@@ -149,22 +152,30 @@ CREATE TABLE t1 (
 You can directly attach the default placement options to a database schema. This works similar to setting the default character set or collation for a schema. Your specified placement options apply when no other options are specified. For example:
```

Comment:
Refer to https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR151.
```suggestion
You can directly attach the default placement rules to a database schema. This works similar to setting the default character set or collation for a schema. Your specified placement options apply when no other options are specified. For example:
```